src/ResourceTemplateValidator.js

```diff
@@ -36,7 +36,7 @@ const validateRepeatedPropertyTemplates = (resourceTemplate) => {
     }
   })
   if (dupes.length > 0) {
-    return [`Repeated property templates with same property URI (${dupes}) are not allowed.`]
+    return [`Validation error for ${resourceTemplate.resourceURI}: Repeated property templates with same property URI (${dupes}) are not allowed.`]
```

Comment:
The string `Validation error for ${resourceTemplate.resourceURI}` is repeated multiple times in the `ResourceTemplateValidator` module and could be a variable that is defined once

Reply:
Fixed.